spec/javascripts/components/story/expanded_story/expanded_story_spec.js

```diff
@@ -4,19 +4,64 @@ import { ExpandedStory } from 'components/story/ExpandedStory/index';
 import storyFactory from '../../../support/factories/storyFactory';
 
 describe('<ExpandedStory />', () => {
+  const defaultProps = {
+    story: storyFactory(),
+    editStory: sinon.spy(),
```

Comment:
Are we using these spies in assertions? If so, we need to re-create them for each test.